charts/argo/Chart.yaml

```diff
@@ -1,5 +1,5 @@
 apiVersion: v1
-appVersion: "v2.7.6"
+appVersion: "v2.7.7"
```

Comment:
No need to change the `appVersion`, the correct value here is still `v2.7.6`.

Reply:
fixed